src/test/java/core/basesyntax/service/impl/ReportServiceImplTest.java

```diff
@@ -0,0 +1,68 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ReportService;
+import core.basesyntax.service.TransactionException;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class ReportServiceImplTest {
+    private static final String EXCEPTION_MESSAGE
+            = TransactionException.class.toString();
+    private static final String HEADER = "fruit,quantity";
+    private static final String SEPARATOR = System.lineSeparator();
+    private static ReportService reportService;
+
+    @BeforeAll
+    static void beforeAll() {
+        reportService = new ReportServiceImpl();
+    }
+
+    @Test
+    void report_emptyStorage_notOk() {
+        Assertions.assertThrows(TransactionException.class, () -> reportService.createReport(),
+                String.format("%s should throw for empty storage", EXCEPTION_MESSAGE));
+    }
+
+    @Test
+    void report_isNotEmptyStorage_Ok() {
+        Storage.fruits.put("banana", 20);
+        Storage.fruits.put("apple", 10);
+        Assertions.assertNotEquals(HEADER, reportService.createReport());
```

Comment:
don't create negative test here, the fact that report isn't just header tell us little about it behaving correctly. Maybe the report is actually "Encountered unexpected error on line 47" or whatever